pkg/logs/input/docker/tailer.go

```diff
@@ -190,6 +189,10 @@ func (t *Tailer) readForever() {
 					// of the tailer, stop reading
 					return
 				case isContextCanceled(err):
+					// It could happen that the docker daemon takes a lot of time gathering timestamps
+					// before starting to send any data when it has stored several large log files.
+					// Increasing the docker_client_read_timeout could help avoiding such a situation.
+					log.Warnf("docker client read timeout (after %s) in the tailer", t.readTimeout.String())
```

Comment:
I don't think we should log a warning here, we should keep this in the debug-level log that's logged by `tryRestartReader`.

The reason is that there are other circumstances where we actually expect a read timeout and shouldn't log a warning (IIRC, can happen when `max-file` is set to `1`  in the docker logging driver and a file rotation is performed)

Reply:
Gotcha, I've just kept the comment then.